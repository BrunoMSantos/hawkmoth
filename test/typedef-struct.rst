
.. c:type:: struct named

   named typedeffed struct


   .. c:member:: int m

      named member


.. c:type:: typedef_struct

   unnamed typedeffed struct


   .. c:member:: int m

      unnamed member

