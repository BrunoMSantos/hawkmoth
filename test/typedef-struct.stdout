.. c:type:: struct named

   
   named typedeffed struct
   

.. c:member:: int named.m

   
   named member
   

.. c:type:: typedef_struct

   
   unnamed typedeffed struct
   

.. c:member:: int typedef_struct.m

   
   unnamed member
   

